modules/programs/neovim.nix

```diff
@@ -132,15 +140,6 @@ in {
         '';
       };
 
-      generatedConfigViml = mkOption {
-        type = types.lines;
-        visible = true;
-        readOnly = true;
-        description = ''
-          Generated vimscript config.
-        '';
-      };
-
```

Comment:
Was this removed by mistake?

This option allows people to use the generated neovim configuration in other ways.

Reply:
Sorry, I didn't thought about other possible usages of this attribute other than setting specific things order in init.vim.
Gonna add it back.

Reply:
A doubt: Should this generatedConfigViml now include also the `initExtra` attribute?